treeherder/webapp/api/urls.py

```diff
@@ -38,6 +39,18 @@
 )
 
 project_bound_router.register(
+    r'seta/v1/job-priorities',
```

Comment:
The API has support for versioning (http://www.django-rest-framework.org/api-guide/versioning/) so I don't think we need to hardcode a version in the URL here?